### DISCUSSION TITLE
Add element/row above table

### DISCUSSION THREAD

**Discussion by bhaloditarun** [QuestPDF/QuestPDF#570] [General]
Hello @MarcinZiabek 

First of all, I know there is spacing property and paddingtop and showonce() properties but its not working for my requirements.

My table is always display at top and if add paddingtop then display in all pages.

**I want to add element/row at top and ONLY for first page.**

Is there any way we can add rows inside table(before table column name) or nested table etc...

My pdf is display as below:
![quest-pdf](https://github.com/QuestPDF/QuestPDF/assets/58458934/90775055-b52d-498f-9f26-955836bb19a9)

Let me attach my code so, get more idea:
[code.txt](https://github.com/QuestPDF/QuestPDF/files/11555464/code.txt)

Thank you in advance,
Tarun

**Comment by MarcinZiabek** [comment]
You can merge content with simple elements such as Column:

```csharp
.Page(page =>
{
    page.Margin(1, Unit.Inch);
    
    page.Content()
        .Column(column =>
        {
            // display something only only once
            column.Item().Height(50).Background(Colors.Black);
            
            // display table or any other pageable content, e.g. table
            column.Item().Table(table =>
            {
                table.ColumnsDefinition(definition =>
                {
                    definition.RelativeColumn();
                    definition.RelativeColumn();
                    definition.RelativeColumn();
                });

                foreach (var i in Enumerable.Range(0, 50))
                {
                    table
                        .Cell()
                        .Background(Placeholders.BackgroundColor())
                        .Height(50)
                        .AlignCenter()
                        .AlignMiddle()
                        .Text(i.ToString())
                        .FontSize(18);
                }
            });
        });
});
```

That results with:

<img width="1196" alt="image" src="https://github.com/QuestPDF/QuestPDF/assets/9263853/10cba1e6-bef9-446e-888b-3c4b323ed3d9">
